src/local-models/Llama3/README.md

```diff
@@ -0,0 +1,182 @@
+# Llama 3 using ollama
+
+This repo contains a sample on how to use a llama 3 model with semantic kernel.
+
+## Using Local LLMs like Llama3 or Phi-3
+
+Local LLMs like Llama3 and Phi-3 represent a significant shift towards more efficient and compact AI models. 
+
+- [**Llama3**](https://llama.meta.com/llama3/), with its Mixture-of-Experts (MoE) architecture, offers specialized neural networks for different tasks, providing high-quality outputs with a smaller parameter count. 
+- [**Phi-3**](https://azure.microsoft.com/en-us/blog/introducing-phi-3-redefining-whats-possible-with-slms/), developed by Microsoft, uses advanced training techniques like quantization to maximize efficiency, making it ideal for deployment on a wide range of devices.
+
+The use of local LLMs offers several advantages:
+
+- **Reduced Latency**: Local models eliminate network latency associated with cloud-based solutions.
+- **Enhanced Privacy**: Data remains on your local device, offering a secure environment for sensitive information.
+- **Customization**: Local models allow for greater flexibility to tweak and optimize the models as per your needs3.
+
+## Prerequisites
+
+To run Phi-3 or Llama3 model on your local machine, the following prerequisites are required:
+
+- [Install ollama](https://ollama.com/download). Ollama can be installed in macOS, Linux or Windows. 
+- Download and run a LLM. 
+    [Ollama library](https://ollama.com/library) provides different types of open source models. To install Llama3 run the command:
+
+    ```bash
+    ollama run llama3
+    ```
+- The command `ollama run phi3` will install Phi-3.
+
+- The command `ollama list` will show the local installed models:
+
+    ![Image Description](img/10ollamarunllama3.png)
+
+***Tip**: To see the ollama service journal in realtime, you can run the command:*
+
+```bash
+journalctl -u ollama -f
+```
+
+
+## Quick Start, using Llama3 model as an example
+
+The llama3 model is locally running, we can now run some prompts using C# and Semantic Kernel.
+
+### Step 1: Run the model
+
+The file `src/local-models/Llama3/Program.cs` shows an example on how to use Semantic Kernel to use the local model. When adding the OpenAI Chat Completion service to the kernel, we must specify the modelId, endpoint and apikey (usually an empty string).
```

Comment:
Consider renaming the Llama3 directory since it has both Llama 3 and Phi-3

Reply:
Got it! I'll rename it to phi3-llama3